docs/get-started/ide/phpstorm/debugging.md

```diff
@@ -0,0 +1,142 @@
+Debugging
+========
+
+## Using MAMP and Xdebug
+
+This section shows you on how to set up your environment in MAMP and PhpStorm to debug your Joomla! extensions.
+It is assumed that PhpStorm is in the current version and that you have a local Joomla! installation in which your
+extension is already installed.
+
+### Enable Xdebugger in MAMP (Pro)
+
+In MAMP open the Settings and switch to the Languages page. Select the PHP Tab and make sure that Xdebug is ticked in
+the Extensions section.
+
+![MAMP PHP Language Settings](./assets/mamp_language_settings.png)
+
+### Add IDE Key to your php.ini
+
+Edit php.ini Template inside MAMP, search for the xdebug section and add:
+
+```ini php.ini
+xdebug.idekey=PHPSTORM
+```
+
+![MAMP PHP INI Settings](./assets/php_ini_setup.png)
+
+Hint: Use `Ctrl` + `F` and search for "xdebug"
+
+:::note
+
+After you have changed some settings make sure to save & restart your MAMP Server.
+
+:::
+
+### Check PHP Configuration in your Joomla! Installation
+
+Check in the PHP Settings in your Joomla! Backend in `System` > `System Information` > `PHP
+Information` and make sure that the Section for Xdebug is visible:
+
+![Joomla! PHP Information](./assets/xdebug_top.jpg)
+
+### Install Xdebug Browser Extension
+
+Install a Xdebug Extension of your choice in your Browser. Search for Xdebug in your Browser's Extensions Dictionary
+and enable the Plugin - Check your Browser's Guide on how to install Plugins. Please note: An installation may require
+a Browser restart.
+
+### Setting up Environments in PhpStorm
+
+Mostly you will write your Code in its own Project Folder but when using Joomla! The project will be zipped / installed
+or deployed from inside your IDE into to your Joomla Dev Environment. Which means you cannot use debugger directly
+inside the Project where you write your extension code (because it is not the file which will be run by Joomla!
+normally).
+
+So beside your extension development project in PhpStorm open or create a new Project for Debugging inside your local
+Joomla! Installation. This Project should point to the Joomla! Main directory of your local Joomla! Installation. So in
+PhpStorm click on `File` > `New Project form Existing Files`and select your Joomla! Installation. PhpStorm may ask you
+if you would like to Trust this Folder, confirm that request.
+
+### Configure PhpStorm Debugger
+
+In this new Project select `Run` from the Main Menu and Select `Edit Configurations`.
+
+![Edit Configurations Menu](./assets/run_edit_configurations.png)
```

Comment:
```suggestion
![Edit Configurations Menu](./assets/run_edit_configurations.png)
```

Please move all images from "assets" to "_assets" folder

Reply:
done